### ISSUE TITLE
Assignment 1 Tim Lugtenburg

### ISSUE THREAD

**Issue by TimTheChemist123** [EstherPlomp/TNW-RDM-101#78]
### Introduction

Hi everyone, my name is Tim Lugtenburg and I am a PhD candidate in the field of Biocatalysis (TNW faculty). 

### Describe your research in 2-3 sentences to someone that is not from your field (please avoid abbreviations)

In my research my aim is to engineer novel biocatalysts (enzymes) that are able to perform oxyfunctionalization  reactions more efficiently. Improving these types of synthesis reactions is important in chemistry for the more sustainable and/or profitable production of e.g. pharmaceuticals, flavour compounds or fragrances. 

### My research entails the following aspects: 

| Research Aspect | Answer |
| ------------- | ------------- |
| Use/collect personal data (health data, interviews, surveys)  | No  |
| Use/collect experimental data (lab experiments, measurements with instruments)  | Yes  |
| Collaborate with industry  | No  |
| Write/develop software as the main output of the project  | No  |
| Use code (as in programming) for data analysis | No  |
| Work with large data (images, simulation models) | No  |
| Other:  | N/A  |
### Reflections on the importance of RDM videos

I very much agreed with the importance of having a well structured organization of research data, not only for your personal use, but even more importantly that it can be communicated to others. A horror story for me would be to reach a certain early stage conclusion and basing the rest of the research on that, but later on not being able to defend your decisions as the data is not findable anymore. This could be due to either improper labelling, but part of the difficulty is knowing which aspects of the experiment are relevant to write down, as this is not always clear at the start.

### What would you like to learn during this course? 

Ideally I would like to learn more about useful workflows for the organization of the data specific to my field of research. As I will not be working with a lot of code like other fields, nor with large datasets. However, the data I will produce consists of raw data from experiments in all kinds of formats, which are turned into graphs. Better organization of this data where I know where to store what, in a logical way, that does not require too much administration would be very helpful to me. Another thing I would like to discuss is the use of productivity tools like Notion for planning of my research, and whether the use of a 3rd party app like that does not violate privacy regulations of the TU.

### Checklist assignments 

- [x] Assignment 1: creating a GitHub issue (before **12 September, 10:00**)
- [x] Respond to the GitHub issue #65 
- [x] [Assignment 2](https://estherplomp.github.io/TNW-RDM-101/06-Assignment-Data-Flow-Map-1.html): Data Flow Map 1 (share a link in this issue before **20 September 13:00**). Link: https://surfdrive.surf.nl/files/index.php/s/HYOsMlQoIhd9LRr
- [x] Provide feedback to at least one Assignment 2 from another participant
- [x] Respond to the GitHub discussion #66
- [x] Respond to the GitHub discussion #67
- [x] [Assignment 3](https://estherplomp.github.io/TNW-RDM-101/11-Assignment-Data-Flow-Map-2.html): Data Flow Map 2 (share a link in this issue before **4 October 13:00**). Link: 
https://surfdrive.surf.nl/files/index.php/s/PtWSHcGf3h14QGj
- [x] Provide feedback to at least one Assignment 3 from another participant
- [x] [Assignment 4](https://estherplomp.github.io/TNW-RDM-101/13-Assignment-DMP.html): Data Management Plan (before **11 October, 13:00**)
- [x] Respond to the GitHub discussion #68 if you have any questions (optional)
- [x] [Assignment 5](https://estherplomp.github.io/TNW-RDM-101/14-Assignment-Data-Flow-Map-3.html): Data Flow Map 3: submit your slide (before **17 October, 17:00**)

**Comment by EstherPlomp** [comment]
Hi @TimTheChemist123! I'll respond here to your question on the use of third party apps: 

>>
> Another thing I would like to discuss is the use of productivity tools like Notion for planning of my research, and whether the use of a 3rd party app like that does not violate privacy regulations of the TU.

This depends a bit on your research project and situation - privacy regulations are not physics unfortunately... Since you're not dealing with personal data and/or company data, you have less restrictions on what solutions you could use compared to other colleagues. In principle you can use these tools as long as you don't use them to process any sensitive information. You can argue that your name/email address is not that sensitive, as I can also access this personal data from the public TU Delft website. If you start adding people's birthdates in Notion it is a different story - but I doubt you'll need that information to be able to plan your research :). If you have any doubts about other tools, please let me know!

In short: you can use these tools, just be conscious about what type of data you're storing in them!

**Comment by TimTheChemist123** [comment]
Hi!

Here is my assignment 2:
https://surfdrive.surf.nl/files/index.php/s/HYOsMlQoIhd9LRr 

-Tim

**Comment by RiccardoDalMas** [comment]
Hi @TimTheChemist123,

Very nice and thorough data flow map!

I was wondering about how you plan to handle the transfer of the raw data generated by the instruments. You mention that the GC data are stored on the instrument itself, is it the case also for the other instruments you'll use (NMR, UV VIS, etc.)? If so, will you make a copy on the project drive of the raw data, and can you do this directly from the local terminal connected to the instrument?

Thank you,
Cheers,
Riccardo

**Comment by Lloyd537** [comment]
I like your data flow map, very extensive! Especially the distinction between data that is relevant for the project and others, and data (notes, etc) that is only relevant to you. I do the same to some degree, I have some note documents that I use for my own project planning. While I agree that these are not relevant to others, it might still be a good idea to back these up as well, since you do need them to continue your project.

Cheers,
Lloyd

**Comment by EstherPlomp** [comment]
Thanks for sharing assignment 2 @TimTheChemist123! It looks very clear and comprehensive - well done!

- As @Lloyd537 and @cfprimavera already mention: Indeed make sure that this administrative data is backed up - you don't want to loose your manuscript after you almost finished it! The online personal drive (H:), OneDrive and SURFdrive could all be solutions for this
- I agree with @RiccardoDalMas comment here: do check if the raw data is backed up as well - if it is not backed up at the instrument, make sure you save a copy somewhere. 
- I don't see any data sizes indicated - I think you won't reach TBS of data, so hopefully you don't run into storage shortages :) 
- I like your approach of having a physical notebook and then transferring the relevant information to a digital notebook!
- Great use of the red flags! Indeed, sometimes just forgetting to do something can be your worst enemy...
-  For some of the datasets it is not entirely clear to me what the file formats would be - this becomes more important for assignment 3!
- I also like how you indicated that you have qualitative data as you have lab observations! That is indeed more qualitative and subjective sometimes

**Comment by TimTheChemist123** [comment]
Here is my assignment 3:
https://surfdrive.surf.nl/files/index.php/s/PtWSHcGf3h14QGj

-Tim

**Comment by EstherPlomp** [comment]
Thanks for sharing assignment 3 @TimTheChemist123! It again looks fantastic - well done! I therefore have very little feedback and just some thoughts: 

 **Data Organisation**
- good point about the moving of files!

**File Formats**
- No worries about Mstrenova files not being open - if it is indeed the most commonly used file format in the field it should still be reusable for people. Ideally things are open - but they should also be usable/workable :)

**Comment by HuaChenTUDelft** [comment]
Hi Tim,

Really nice data flow map and thanks for sharing with us. I really agree with you that numbering batches of experiments does help to find the corresponding data if you have to change the folder structure in the future. And also name the data with corresponding charateristics also helps to organise it.

Best,
Hua

**Comment by FanJ-TUD** [comment]
Hello Tim,

Very detailed data flow map. I really appreciate how you precisely describe the properties with flags in your map, which quite customize the function for understanding and reminding the exp process.
